zone: 0
hand_speed: 0.99
hand_force: 0.9
tcp: 35.0
ori: 100.0

